style.css

```diff
@@ -0,0 +1,273 @@
+* {
+    font-family: sans-serif;
+}
+body {
+  margin: 0;
+}
+.grid {
+  grid-template-columns: 300px 1fr 400px;
+  grid-template-rows: 100vh;
+}
+.left-bar,
+.right-bar,
+header,
+.text,
+.shadow-gap,
+.content {
+  border: solid 1px rgb(172, 172, 172);
+}
+.g, .grid {
+  display: grid;
+}
+.shadow-gap {
+    background-color: rgb(208, 208, 208);
+}
+.prof-border {
+  margin-left: 5px;
+  margin-bottom: 15px;
+  border: solid 1px black;
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+  overflow: hidden;
+}
+.profile-pic {
+  width: 100px;
+  position: relative;
+  bottom: 17px;
+  right: 30px;
+}
+.legollas {
+    width: 70px;
+    bottom: 25px;
+    right: 20px;
+}
+.legolas-image {
+    width: 80%;
+}
+.gandalf {
+    right: 41px;
+    margin-bottom: 10px;
+}
+#refresh {
+    justify-content: end;
+    padding-right: 5px;
+    padding-top: 5px;
+}
+.left-bar {
+  justify-content: end;
+  grid-template-rows: 1fr 8fr 1fr 2fr;
+}
+.left-bar > div:nth-child(1) > img, .prof-border {
+  padding-top: 10px;
+  padding-left: 10px;
```

Comment:
A variety of approaches for selecting elements used throughout; displaying a good understanding of Speciifitiliity